go/mysql/conn.go

```diff
@@ -36,14 +36,12 @@ import (
 	"vitess.io/vitess/go/vt/vterrors"
 )
 
+// Constants for how ephemeral buffers were used for reading / writing.
```

Comment:
This change broke the build in a scary fashion. Looks like you can't combine const blocks that use iota with other const variables.

Reply:
I need to reset this file. Hold on.